doc/README.md

```diff
@@ -1,7 +1,29 @@
 # Installation for OSX Development
+TODO(raj): Write a script to do all this

```

Comment:
We wouldn't necessarily need to do it for a specific development environment if it is dockerized.

Reply:
resolved to containerization TODO